packages/terra-profile-image/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ## 3.41.1 - (January 4, 2022)
 
+* Changed 
+  * When `src` is not provided Aria-hidded is true.
```

Comment:
Please change this so consumer can understand the impact of the change and move this to unreleased section.